integration-cli/docker_api_containers_test.go

```diff
@@ -841,3 +841,34 @@ func (s *DockerSuite) TestStartWithTooLowMemoryLimit(c *check.C) {
 	c.Assert(status, check.Equals, http.StatusInternalServerError)
 	c.Assert(strings.Contains(string(b), "Minimum memory limit allowed is 4MB"), check.Equals, true)
 }
+
+func (s *DockerSuite) TestContainerApiRename(c *check.C) {
+	runCmd := exec.Command(dockerBinary, "run", "--name", "first_name", "-d", "busybox", "sh")
+	out, _, err := runCommandWithOutput(runCmd)
+	if err != nil {
+		c.Fatal(err)
+	}
+
+	containerID := strings.TrimSpace(out)
+	newName := "new_name" + stringid.GenerateRandomID()
+	statusCode, _, err := sockRequestRaw("POST", "/containers/"+containerID+"/rename?name="+newName, nil, "")
+	if err != nil {
+		c.Fatal(err)
+	}
+
```

Comment:
with the new gocheck pkg in the test suite we can now use `c.Assert` like done here https://github.com/docker/docker/blob/c7845e27ee2b0462b01d043caeb771f21d5e4ac7/integration-cli/docker_api_containers_test.go#L32